docs/Installing_a_Chef_workstation.md

```diff
@@ -6,56 +6,29 @@ These instructions cover commissioning a Chef client node on an EC2 server as pa
 * A web-browser with which you can visit the Chef server Web UI.
 
 ## Create the instance
 
-Create a `m1.small` AWS EC2 instance running `Ubuntu Server 12.04.1 LTS` using the AWS web interface.  Configure its security group to allow access on port 22 (for SSH). The SSH keypair you provide here is referred to below as `<amazon_ssh_key>`. It is easiest if you use the same SSH keypair for all of your instances.
+Create a `t2.micro` AWS EC2 instance running `Ubuntu Server 14.04.2 LTS` using the AWS web interface.  Configure its security group to allow access on port 22 (for SSH). The SSH keypair you provide here is referred to below as `<amazon_ssh_key>`. It is easiest if you use the same SSH keypair for all of your instances.
 
-Configure a DNS entry for this machine, `chef-client.<zone>`. (The precise name isn't important, but we use this consistently in the documentation that follows.) It should have a non-aliased A record pointing at the public IP address of the instance as displayed in the EC2 console.
+Configure a DNS entry for this machine, `chef workstation.<zone>`. (The precise name isn't important, but we use this consistently in the documentation that follows.) It should have a non-aliased A record pointing at the public IP address of the instance as displayed in the EC2 console.
 
 Once the instance is up and running and you can connect to it over SSH, you may continue to the next steps.
 
 If you make a mistake, simply delete the instance permanently by selecting "Terminate" in the EC2 console, and start again. The terminated instance may take a few minutes to disappear from the console.
 
-## Prepare APT
-
-Connect to `chef-client.<zone>` as the `ubuntu` user over SSH.
-
-Install the `add-apt-key` tool.
-
-    sudo apt-get install add-apt-key -y
-
-Under sudo, create `/etc/apt/sources.list.d/opscode.list` with the following content:
-
-    deb http://apt.opscode.com/ precise-0.10 main
-
-Install the GPG key for this repository:
-
-    sudo add-apt-key 83EF826A
-
-Install the Chef keyring and update APT's indexes.
-
-    sudo apt-get update
-    sudo apt-get install opscode-keyring -y
-    sudo apt-get upgrade -y
-
-Once this is done, you can continue on to install the Chef client software.
-
-## Install the Chef client software
+## Install Ruby 1.9.3
 
-Install the Chef client tool.
+The Clearwater chef plugins use features from Ruby 1.9.3.  To start run the following command.
 
-    sudo apt-get install chef -y
+    curl -L https://get.rvm.io | bash -s stable
 
-During the install you will be prompted for the Chef server URL, enter `http://chef-server.<zone>:4000`.
+This may fail due to missing GPG signiatures. If this happens it will suggest a command to run to resolve the problem (e.g. gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3`). Run the command suggested, then run the above command again, which should now succeed).
 
-## Install Ruby 1.9.3
+Next install the required ruby version.
```

Comment:
If we use the system ruby we have to `sudo` all the following commands.  RVM will also install 1.9.3 with the latest security patches (all 551 of them!).

This also reminds me that 1.9.3 went EOL in February and we should really be moving people to 2.2.0 at some point (which would require RVM or similar to manage).

I vote we stick with RVM for now.